src/components/FloatingContainer/index.js

```diff
@@ -149,6 +150,10 @@ function FloatingContainer(props) {
       }
       container.style.maxWidth = `${window.innerWidth - targetPosition.left - target.offsetWidth - offset}px`;
     }
+
+    if (parseFloat(container.style.left.replace('px', '')) < 0) {
+      container.style.left = '5px';
+    }
```

Comment:
Hmm, looks more like a hack fix. It shouldn't end up in the situation where this `if` is executed.
I will have a look at it at some point and see why it happens in the first place.

Reply:
the thing is this formula 
`${targetPosition.left + target.offsetWidth / 2 - container.offsetWidth / 2}px` can give negative result

Reply:
with the right edge we don't have a problem because we're setting all the time the `left` property